components/eam/cime_config/testdefs/testmods_dirs/eam/cbool/shell_commands

```diff
@@ -0,0 +1,3 @@
+if [ `./xmlquery --value COMPILER` == ibm ]; then
+  ./xmlchange --force --append CAM_CONFIG_OPTS='-cppdefs -DUSE_CBOOL'
+fi
```

Comment:
Why does this need to be a test modifier? 
Shouldn't it just be part of the compiler configuration?

Reply:
After discussing on Slack, the `USE_CBOOL` cpp macro will be added to config_compilers.xml instead of a testmod.